src/index.d.ts

```diff
@@ -284,6 +284,11 @@ declare namespace RamdaAdjunct {
          */
         stubNull():  null;
 
+        /**
+         * A function that returns new empty array on every call.
+         */
+        stubArray():  {};
```

Comment:
Shouldn't we infere type here, not the value ?
```javascript
stubArray(): Array;
```

Reply:
oops, must be `[]`, literal types

Reply:
Will you change for literal type `[]` ? After that we can merge.

Reply:
Oops))))